src/__tests__/axeReporterEarl.test.ts

```diff
@@ -0,0 +1,161 @@
+import { assert } from 'chai'
+// @ts-ignore:
```

Comment:
Why are you doing this?

We should all forget that this directive exists. If we want the benefits of TS, we need to allow the compiler to do its job. Turning it off causes nasal demons and prevents us from reaping all of TS' goodness.

Reply:
Because jsonld doesn't have a type definition. Any suggestions on an alternative?

Reply:
This is the *only* thing I don't love about TS. Of the package you're using doesn't ship its own definitions, and they don't exist in the `@types` org, you have to write your own. For example:

https://github.com/dequelabs/attest-node-suite/blob/develop/packages/attest-webdriverio/src/types/chromedriver.d.ts

Once you do this, the compiler will find the definitions and you'll be able to use your package normally.

Reply:
There seems to be interfaces for `jsonld` - https://github.com/types/jsonld

A good place to check for types is definitely typed - http://definitelytyped.org/